src/coffeeandtacos/CoffeeAndTacos.java

```diff
@@ -25,33 +25,29 @@ public static void main(String[] args) {
         logger.logInfo("File Parsed");
         // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
         // HINT:  You'll need two nested forloops
```

Comment:
Remove unnecessary comments. Any instructions from the actual problem can be removed. The only comments that should remain are instructional ones for yourself or others when you look back on the code.